openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -985,9 +996,16 @@ def _stream_frag_discr_data(curr, func_obj, is_first):
         # of curves
         desc.text = 'Fragility from GVD: %s' % func_obj.name
 
```

Comment:
And inside that function, add an explanatory comment like the one in the changelog